blockly-core/blocks/functional_procedures.js

```diff
@@ -496,6 +496,11 @@ Blockly.Blocks.functional_pass = {
     }
 
     this.setFunctional(true);
+    // functional_pass blocks are immovable, unless we're level editing
+    this.setMovable(!!Blockly.editBlocks);
+    this.setColorFromName_();
+    this.blockSpace.events.listen(Blockly.BlockSpace.EVENTS.BLOCK_SPACE_CHANGE,
```

Comment:
Could try to make this more efficient than running on every block space change, but there's already a precedence for this, and I don't really want to spend a bunch of time potentially optimizing prematurely.